test/unit/instrumentation/http/http.test.js

```diff
@@ -322,6 +322,35 @@ test('built-in http module instrumentation', (t) => {
       }
     )
 
+    t.test('when url_obfuscation regex pattern is set, obfuscate segment url attributes', (t) => {
```

Comment:
can we add a few more tests. I get this is just compiling regex but something more realistic to the use cases of customers.  I'd love to see one that replaces both  `/api/v1/users/112324423/edit` and `/api/v1/tokens/adsfasf12312321` with `/api/v1/users/***/edit` and `/api/v1/tokens/***` respectively.

Reply:
n/m i see something to this effect below